Readme.md

```diff
@@ -7,7 +7,7 @@
 $ helm repo add prefect https://prefecthq.github.io/prefect-helm
```

Comment:
the repo name not math, prefect of prefecthq?

Reply:
You mean line 7? `$ helm repo add prefect https://prefecthq.github.io/prefect-helm` 

I kept that line the same if that's what you refer to

Reply:
if you don't change line 7, and 8, and then line 9 needs to be:

`helm install my-release prefect/<chart>`